README.rst

```diff
@@ -47,7 +47,7 @@ The latest stable version can always be installed or updated via pip:
 
 .. code-block:: bash
 
-    $ pip install --update holidays
+    $ pip install holidays
```

Comment:
```suggestion
    $ pip install --upgrade holidays
```

Reply:
That might be the case, but why would you want the user to force an upgrade if already installed?